tutorials/quickstart/Solution/Platform/Platform.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <LangVersion>7.1</LangVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Particular.PlatformSample" Version="1.*" />
```

Comment:
Ditto

Reply:
_Especially_ for the platform sample, being on latest means getting the latest ServiceControl, Monitoring, and ServicePulse.